glottolog3/models.py

```diff
@@ -541,6 +541,39 @@ class LanguoidSchema(Schema):
     def make_languoid(self, data):
         return Languoid(**data)
 
+def validate_id_type(type):
+    if (type != IdentifierType.iso.value and \
+        type != IdentifierType.glottolog.value and \
+        type != IdentifierType.wals.value and \
+        type != IdentifierType.ethnologue.value and \
+        type != 'name'):
```

Comment:
cleaner way to do this is to make an array for all valid types and check if type not in the array. Just nitpicking.